recipes/default.rb

```diff
@@ -24,6 +24,6 @@
   post_install_action node['chef_client_updater']['post_install_action']
   download_url_override node['chef_client_updater']['download_url_override'] if node['chef_client_updater']['download_url_override']
   checksum node['chef_client_updater']['checksum'] if node['chef_client_updater']['checksum']
-  upgrade_delay node['chef_client_updater']['upgrade_delay'].nil? ? 30 : node['chef_client_updater']['upgrade_delay']
+  upgrade_delay node['chef_client_updater']['upgrade_delay'].nil? ? 61 : node['chef_client_updater']['upgrade_delay']
```

Comment:
This seems very odd that we do not set a default directly in https://github.com/chef-cookbooks/chef_client_updater/blob/master/attributes/default.rb#L42 and instead do this defensive coding.

Reply:
And it really needs to be changed here:   https://github.com/chef-cookbooks/chef_client_updater/blob/master/resources/default.rb#L38

Reply:
`upgrade_delay node['chef_client_updater']['upgrade_delay'] unless node['chef_client_updater']['upgrade_delay'].nil?` is probably what this line of code should be.